src/components/ui/label.tsx

```
@@ -0,0 +1,26 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)